securedrop/journalist_templates/base.html

```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{ g.organization_name }}</title>
+  <title>{% block tab_title %}{% endblock %} | {{ g.organization_name }}</title>
```

Comment:
Is it your intention that this block be [`required`] for all templates?  If not, then we should check for its absence here, to avoid rendered templates like:

```html
<title>| SecureDrop</title>
```

[`required`]: https://jinja.palletsprojects.com/en/3.0.x/templates/#required-blocks

Reply:
My intention was for it to be mandatory, as my understanding of the relevant guideline is that all pages should have some sort of descriptive title. Is this a problem?

Reply:
No, I agree!  In that case you can add the [`required`] attribute to the `tab_title` block so that a template that doesn't override it will raise a `TemplateRuntimeError`.

[`required`]: https://jinja.palletsprojects.com/en/3.0.x/templates/#required-blocks

Reply:
Oh yes, I added it in the new commit. Thanks for the tip!

Reply:
Thanks!